src/main/resources/ath-container/Dockerfile

```diff
@@ -33,9 +33,9 @@ ENV PATH="$PATH:/opt/maven/bin"
 ENV SHARED_DOCKER_SERVICE true
 
 # Allow injecting uid and git to match directory ownership
-ARG uid=1000
+ARG uid=1001
 ENV uid $uid
-ARG gid=1000
+ARG gid=1001
```

Comment:
Why do these need to be changed if you are overriding them anyway?

Reply:
It's related to the condition in the shared library: https://github.com/jenkins-infra/pipeline-library/blob/master/vars/runATH.groovy#L98 .

Based on the logs I see on the failing job (`11:46:50  No building ATH docker container image. Using jenkins/ath:acceptance-test-harness-1.80 as specified` in https://ci.jenkins.io/job/Core/job/jenkins/job/PR-5575/5/consoleFull), I understand that the Docker image is not always build depending on some metadatas provided to the build (either a file or a variable).

That means that the 2nd case should be covered.

Reply:
It's a short-term thing: use the new UID set up in the packer images. I see the 2 following upcoming changes:

- Easy to do, quick win: Adding a routine in jenkins-infra/packer_images that would run on master branch and open a PR here to update the UID if it is changed. Only infra changes, and it would avoid knowledge to be lost in the future.
- Harder but improved ATH: always building the image, using an efficient caching (Docker buildX + `--cache-from` flag) so the layers could be cached, but the "create user step“ could be run all the time.

Reply:
If you are doing anything with UIDs you are probably doing it wrong.